### DISCUSSION TITLE
Removing Ant Media Domain After Changing to Custom Domain

### DISCUSSION THREAD

**Discussion by DeltaCharlie69** [ant-media/Ant-Media-Server#6743] [Q&A]
Hello,

I recently set up Ant Media Server (Enterprise Edition 2.11.3) on a virtual machine, and initially, I configured SSL using Ant Media Server’s domain. I have since switched to using our own domain, but I’m facing issues removing Ant Media’s default domain. Both domains currently allow me access, but I would like to retain access solely through our custom domain.

I’m very new to this process, so any guidance on how to cleanly remove Ant Media’s domain would be greatly appreciated. Here are a few details about my setup:

	•	Platform: DigitalOcean (I have access to the console)
	•	Current Status: Both domains provide access, but I only need access via our custom domain.

If there are specific files or configurations I need to update to remove Ant Media’s domain, please let me know. Any step-by-step guidance would be especially helpful!

Thank you so much for your help!

**Comment by muratugureminoglu** [comment]
Hi @DeltaCharlie69 

I tested on two domains. First I created a certificate for ams-9805.antmedia.cloud and then for ams-test.antmedia.cloud. 

![image](https://github.com/user-attachments/assets/c2f807aa-bcff-473a-a4b9-7052b318126e)

Here the first domain is currently giving a certificate error. 

![image](https://github.com/user-attachments/assets/a7fe30ab-8309-4cb9-86d0-3e6abf53ae10)

Here, working as expected.

![image](https://github.com/user-attachments/assets/ee6878f9-0ae8-46dd-a8fc-421fc3702a9f)

Ant Media Server will be reached for each domain pointing to the server, to avoid this you can delete unnecessary records in DNS.

If you are not familiar with DNS, contact support[@]antmedia.io and we can check together.

Regards.
